### PR TITLE
DOC-4042: Added note regarding Groovy dependencies

### PR DESCRIPTION
**Pull request by Niamh25**
The Corda Shell has a dependency on Groovy which is not on the public API.

### CODE REVIEW

File: content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ As a developer or node operator, you should upgrade to the [latest released vers
 
 Corda Enterprise Edition 4.9.4 is a patch release of Corda Enterprise focused on resolving issues.
 
+{{< note >}}
+ The Corda Shell has a dependency on Groovy which is not on the public API. With the removal of the shell the groovy dependency is also removed. Any flows dependent on groovy need to add the dependency to the CorDapp, the drivers directory, or remove the use of Groovy.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md:38]:
```suggestion
 The Corda shell has a dependency on Groovy which is not on the public API. With the removal of the shell, the Groovy dependency is also removed. Any flows dependent on Groovy need to add the dependency to the CorDapp, the drivers' directory, or remove the use of Groovy.
```

Comment (nadinequinr3) [inline content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md:38]: This isn't very clear to me. What has changed in this release? Is it that the shell now has a dependency on Groovy? Or that the API no longer has a dependency? Or that the shell is removed?

Comment (Niamh25) [reply content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md:38]: In 4.9 the shell was removed from the product and made as a stand alone product. However the shell had a dependency on Groovy which was never on the public API. As some customers rely on Groovy in their flows the note was added to mention that along with the shell, groovy should also be removed!

Comment (nadinequinr3) [reply content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md]: Ok, so it's not specific to 4.9.4 so probably shouldn't be in that section? Maybe check with the reporter but it probably doesn't make sense to retrospectively add it to the 4.9 section. Since it's a note, you could probably put it at the top of the page maybe?

I'd also suggest rewording it a little to make the note clearer. Something like:

```The Corda Shell, which has a dependency on Groovy, is now a standalone product since Corda Enterprise Edition 4.9. As a result, for any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.
```

**Comment by Niamh25** [comment]
Updating

**Review by yarivus — DISMISSED** [review]
LGTM

File: content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md [hunk 2]
```diff
@@ -134,6 +134,7 @@ As a node operator, you should upgrade to the [latest released version of Corda]
 In this release:
 
 * Corda Shell has been removed to its own repository for improved security. You can now use a standalone shell outside of the node, or from within the node's drivers. You can read more about using the standalone shell [here](./node/operating/shell.html#the-standalone-shell). For information on adding the shell to the node's drivers, see [Upgrading a node to Corda Enterprise Edition 4.9](node-upgrade-notes.html).
+{{< note >}} The Corda Shell, which has a dependency on Groovy, is now a standalone product since Corda Enterprise Edition 4.9. As a result, for any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.{{< note >}}
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md:137]:
```suggestion
{{< note >}} The Corda shell, which has a dependency on Groovy, is a standalone product since Corda Enterprise Edition 4.9. As a result, for any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.{{< note >}}
```
Shell is not capitalised in the currect documentation: https://docs.r3.com/en/platform/corda/4.9/enterprise/node/operating/shell.html#ways-of-using-the-corda-shell

File: content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md [hunk 3]
```diff
@@ -134,6 +134,7 @@ As a node operator, you should upgrade to the [latest released version of Corda]
 In this release:
 
 * Corda Shell has been removed to its own repository for improved security. You can now use a standalone shell outside of the node, or from within the node's drivers. You can read more about using the standalone shell [here](./node/operating/shell.html#the-standalone-shell). For information on adding the shell to the node's drivers, see [Upgrading a node to Corda Enterprise Edition 4.9](node-upgrade-notes.html).
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md:136]:
```suggestion
* Corda shell has been removed to its own repository for improved security. You can now use a standalone shell outside of the node, or from within the node's drivers. For more information about using the standalone shell, see [The standalone shell](./node/operating/shell.html#the-standalone-shell). For information on adding the shell to the node's drivers, see [Upgrading a node to Corda Enterprise Edition 4.9](node-upgrade-notes.html).
```
Do not use words such as "here", "click here", and so on as links. Use more meaningful links.

**Review by yarivus — DISMISSED** [review]
LGTM

File: content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md [hunk 4]
```diff
@@ -134,6 +134,7 @@ As a node operator, you should upgrade to the [latest released version of Corda]
...
+{{< note >}} The Corda shell, which has a dependency on Groovy, is a standalone product since Corda Enterprise Edition 4.9. As a result, for any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.{{< note >}}
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md:137]:
```suggestion
{{< note >}} 
The Corda shell, which has a dependency on Groovy, is a standalone product since Corda Enterprise Edition 4.9. As a result, for any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.
{{< /note >}}
```

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md [hunk 5]
```diff
@@ -133,7 +133,10 @@ As a node operator, you should upgrade to the [latest released version of Corda]
 
 In this release:
 
-* Corda Shell has been removed to its own repository for improved security. You can now use a standalone shell outside of the node, or from within the node's drivers. You can read more about using the standalone shell [here](./node/operating/shell.html#the-standalone-shell). For information on adding the shell to the node's drivers, see [Upgrading a node to Corda Enterprise Edition 4.9](node-upgrade-notes.html).
+* Corda shell has been removed to its own repository for improved security. You can now use a standalone shell outside of the node, or from within the node's drivers. For more information about using the standalone shell, see [The standalone shell](./node/operating/shell.html#the-standalone-shell). For information on adding the shell to the node's drivers, see [Upgrading a node to Corda Enterprise Edition 4.9](node-upgrade-notes.html).
+{{< note >}} 
+The Corda shell, which has a dependency on Groovy, is a standalone product since Corda Enterprise Edition 4.9. As a result, for any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md:138]: It looks like this note is now in the 4.9 Release Notes section?
If so, the first sentence is probably redundant as you've said in the previous section that the shell has been separated out and the release affected is implied by the section?

File: content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md [hunk 6]
```diff
@@ -135,7 +135,7 @@ In this release:
 
 * Corda shell has been removed to its own repository for improved security. You can now use a standalone shell outside of the node, or from within the node's drivers. For more information about using the standalone shell, see [The standalone shell](./node/operating/shell.html#the-standalone-shell). For information on adding the shell to the node's drivers, see [Upgrading a node to Corda Enterprise Edition 4.9](node-upgrade-notes.html).
 {{< note >}} 
-The Corda shell, which has a dependency on Groovy, is a standalone product since Corda Enterprise Edition 4.9. As a result, for any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.
+For any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md:138]: I think now you're just missing the context of why they need add the dependency. So something like:

The Corda shell has a dependency on Groovy that the Corda API does not. As a result, for any flows dependent on Groovy, you must now add the dependency to the CorDapp or the drivers' directory.

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md (hunk 6 above)
Comment (Niamh25) [reply content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md]: Updated!

**Review by nadinequinr3 — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM
